capytaine/io/xarray.py

```diff
@@ -281,40 +311,131 @@ def assemble_dataset(results: Sequence[LinearPotentialFlowResult],
 
     # WAVENUMBER
     if wavenumber:
-        dataset.coords['wavenumber'] = wavenumber_data_array(results)
+        if bemio_import:
+            print('Bemio data does not include wavenumber data. wavenumber=True is ignored.')
```

Comment:
But Bemio stores the frequency, the water depth and the gravity constant, am I right?
So it should be relatively easy to support in the future.

Reply:
You are correct... I was so caught up in comparing the `capytaine` and `bemio` structures that I glossed over my basic wave theory! Since you effectively already implemented this in `problems_and_results.py`, this functionality was easy to adapt for `dataframe_from_bemio.py` in commit 0f16541. Thanks for catching this!